samples/storage_csm_unity_v280.yaml

```diff
@@ -79,6 +84,11 @@ spec:
       - name: external-health-monitor
         enabled: false
         args: ["--monitor-interval=60s"]
+      # Uncomment the following to configure how often external-provisioner polls the driver to detect changed capacity
```

Comment:
Please add this block in all the samples that are updated to maintain consistency..  the one under config and the one under e2e

Reply:
done